src/Artsy/Router/index.ts

```diff
@@ -24,16 +25,20 @@ export interface RouterConfig {
      * Defines the type of history to use, depending on router environment.
      */
     protocol?: HistoryProtocol
+
     options?: HistoryOptions
   }
+
   /**
    * Initial route for router on boot
    */
   initialRoute?: string
+
   /**
    * Array of routes to be passed to Found
    */
-  routes: RouteConfig
+  routes: RouteConfig[]
```

Comment:
Do we really need this to be an array for our purposes?

Reply:
Yup, `RouteConfig[]` is correct. I wonder why this didn't get flagged earlier.